app/controllers/ats/positions_controller.rb

```diff
@@ -321,44 +321,42 @@ def position_status_options(position)
   end
 
   def set_side_header_predefined_options
-    @options_for_collaborators =
+    members_query =
       Member
       .includes(:account)
```

Comment:
We have a default_scope in a member model, so maybe this `includes` statement is redundant. And if this `includes` is to be removed then `members_query` variable will only order members, so maybe it should be renamed to something like `ordered_members` or even `alphabetically_ordered_members`. Or removed at all